Explorer/Assets/DCL/InWorldCamera/PhotoDetail/PhotoDetailParameter.cs

```diff
@@ -0,0 +1,25 @@
+using DCL.InWorldCamera.CameraReelStorageService.Schemas;
+using System;
+using System.Collections.Generic;
+
+namespace DCL.InWorldCamera.PhotoDetail
+{
+    public struct PhotoDetailParameter
```

Comment:
Please provide a short description about what's the purpose of the struct, and another description for each public field.